.github/workflows/feed.yml

```diff
@@ -103,7 +103,7 @@ jobs:
           using Twitter
           twitterauth(ENV["API_Key"], ENV["API_Key_Secret"], ENV["Access_Token"], ENV["Access_Token_Secret"])
           
-          myStatus = string("New #JuliaLang package announced: ", ENV["PR_NAME"],  "\n$text \nLearn more: $pr_url")
+          myStatus = string(ENV["PR_NAME"],  " announced #JuliaLang\n$text")
```

Comment:
So what will be the new test now?

Reply:
This should fix the issue with the tweet being too long as well

Reply:
I saw the tweet,  but there is a lot of text, can't easily guess what is what :slightly_smiling_face:

Reply:
"New package: MultiAgentPathFinding v0.1.0" is `ENV["PR_NAME"]` and `text` is everything else. The other parts of the tweet are the text I removed in this PR

Reply:
On Slack we pass the "title" as `ENV["PR_NAME"]` and the body of the message as `text`. It looks like this:
<img width="655" alt="Screen Shot 2022-05-24 at 01 26 37" src="https://user-images.githubusercontent.com/5846501/169920099-6e214ddf-1ecf-4980-8146-3625d89383ab.png">
FWIW

But the "announced" bit is weird- this is for prospective regisrations, NOT new packages. We send these to Slack/zulip at registration time exactly so the community can comment. I think an "announcement" should wait until the package is registered.

Reply:
So actually I think putting a twitter feed here is a bad idea. We should just add another GHA triggered on merges or something, to do this once the package is actually registered

Reply:
I will switch to registered

Reply:
I am not opposed to getting more eyes on these packages before they are merged

Reply:
@ericphanson Would it be as simple as changing the trigger for the Twitter feed to be "merged" instead of "created"?

Reply:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-when-a-pull-request-merges @DilumAluthge yes we could, I don't feel strongly about posting before or after the merge, so I'll defer to @ericphanson

Reply:
I made https://github.com/JuliaRegistries/General/pull/60897 if we want to go the "announce" route

Reply:
I would be careful about auto-tweeting unmoderated user-generated package names and in anyway implying they are endorsed somehow... If they are registered then they at least have gone through a 3 day waiting period and probably do not contain racial slurs in the package name. If not... who knows what people will do esp. if it will bring publicity through twitter.

Reply:
Agreed, let's go with #https://github.com/JuliaRegistries/General/pull/60897

Reply:
Approved there, thanks for the feedback!